commerce_coordinator/apps/lms/filters.py

```diff
@@ -4,22 +4,19 @@
 
 from openedx_filters.tooling import OpenEdxPublicFilter
 
-
-class OrderCreateRequested(OpenEdxPublicFilter):
+class PaymentPageRedirectRequested(OpenEdxPublicFilter):
     """
-    Filter to gather order data from the defined PipelineStep(s)
+    Filter to gather payment page redirect urls.
     """
     # See pipeline step configuration OPEN_EDX_FILTERS_CONFIG dict in `settings/base.py`
-    filter_type = "org.edx.coordinator.lms.order.create.requested.v1"
+    filter_type = "org.edx.coordinator.lms.payment.page.redirect.requested.v1"
 
     @classmethod
-    def run_filter(cls, params, order_data=None):
+    def run_filter(cls, params):
         """
-        Call the PipelineStep(s) defined for this filter, to gather orders and return together
-        Arguments:
-            params: arguments passed through from the original order history url querystring
-            order_data: any preliminary orders we want to append to when running the pipeline
+        Execute the filter pipeline with the desired signature.
         """
 
-        # super op
-        return super().run_pipeline(params=params, order_data=order_data)
+        rollout_data = rollout.pipeline.GetActiveOrderManagementSystem.run_pipeline(params)
```

Comment:
Can't we make this a step in the pipeline as well? Why do we need to execute it explicitly? Can you verify this

Reply:
This is the pipeline created in @JadeyOlivier ticket. Based on the result of her pipeline, that determines which redirect url gets build